packages/botonic-plugin-contentful/src/cms/contents.ts

```diff
@@ -69,7 +69,7 @@ export abstract class TopContent extends Content {
     return new TopContentId(this.contentType, this.id)
   }
 
-  cloneWithFollowUp(newFollowUp: FollowUp): TopContent {
+  cloneWithFollowUp(newFollowUp: FollowUp): this {
```

Comment:
Why wasn't the return type correct before? `TopContent` is the right thing here, no?

Reply:
it's explained here https://www.typescriptlang.org/docs/handbook/advanced-types.html#polymorphic-this-types